src/components/@common/Button/index.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import Container from './styles';
 
-interface PropType {
+interface PropTypes {
```

Comment:
interface/type 은 다른 변수와 혼용되어 사용할 일도 적고, 보통 네이밍을 PascalCase 로 가져가 따로 Types postfix 가 없어도 되지 않을까 생각합니다!
```suggestion
interface Props {
```